misk/src/main/kotlin/misk/perf/PauseDetector.kt

```diff
@@ -40,11 +40,11 @@ internal class PauseDetector @Inject constructor(
   /**
    * Tracks the distribution and total amount of pause time observed.
    *
-   * (Using a summary instead of a histogram to prefer more accurate node-local data over
-   * more accurate aggregations)
+   * (We prefer histogram to summary here because the latter has higher CPU overhead, this runs
+   * ideally at close to 1k QPS, and we don't strictly require accurate quantiles.)
    */
-  private val pauseSummary: Summary =
-    metrics.summary("jvm_pause_time_summary_ms", "Summary in ms of pause time", listOf())
+  private val pauseHistogram: Histogram =
```

Comment:
No code should be depending on this metric, we don't have any official dashboards for pause detection yet, and we have no known external dashboards or monitors making use of this summary. So, I think it should be safe to skip the backwards-compatibility dance here of keeping the old one and having some deprecation period as there is effectively no active usage of this metric other than writing to it.